### PR TITLE
Adiep view empty cart

### PR DESCRIPTION
**Pull request by anndiep-sfcc**

### CODE REVIEW

File: packages/storefront-lwc/src/modules/commerce/basket/basket.js [hunk 1]
```diff
@@ -40,15 +41,12 @@ export default class Basket extends LightningElement {
         return ShoppingBasket.basket.selectedShippingMethodId;
     }
```
Comment (coopaq) [inline packages/storefront-lwc/src/modules/commerce/basket/basket.js]: Interesting: selectedShippingMethodId === hasProducts? I would think `this.products.length`

File: packages/storefront-lwc/src/modules/commerce/basket/basket.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ import { ShoppingBasket } from 'commerce/data';
 
 export default class Basket extends LightningElement {
     @track products = [];
+    @track loading = true;
```
Comment (syedshehrozhussain) [inline packages/storefront-lwc/src/modules/commerce/basket/basket.js]: I dont think we need the @track decorator here.

Comment (anndiep-sfcc) [reply packages/storefront-lwc/src/modules/commerce/basket/basket.js]: You are correct.  Removed.

File: packages/storefront-lwc/src/modules/commerce/basket/basket.js (hunk 1 above)
Comment (anndiep-sfcc) [reply packages/storefront-lwc/src/modules/commerce/basket/basket.js]: This line of code does not exist in the file.

**Pull request merged**